src/ssh/test-sshbridge.c

```diff
@@ -284,6 +286,29 @@ setup (TestCase *tc,
       g_free (content);
     }
 
+  if (fixture && fixture->test_home_ssh_config)
+    {
+      gchar *content;
+
+      host = g_strdup ("some_host");
+      argv[1] = host;
+
+      tc->home_ssh_dir = g_build_filename (tc->home_dir, ".ssh", NULL);
+      g_assert (tc->home_ssh_dir != NULL);
```

Comment:
This duplicates stuff from above, let's just move it further up and do it unconditionally.

Reply:
Oki will do